bolt-design-system/bolt
# Action Blocks | FE | Refactored props to follow conventions

## Jira

http://vjira2:8080/browse/BDS-1151
http://vjira2:8080/browse/BDS-1152

## Summary

Code cleanup on the action blocks component which includes refactoring props to follow Bolt's coding conventions.

## Details

1. Main twig file has been refactored to the latest format.
2. Set up each prop to validate against the schema.
3. Folder structure is re-organized to the latest format.
4. PL pages with action blocks includes have been updated to use new props.

## Deprecation

1. `maxItemsPerRow` has been renamed to `max_items_per_row`.
2. `align` has been updated to `valign` to clearly indicate vertical alignment.
3. `border` has been updated to `borderless` and its default set to `false`.

## How to test

Run the branch locally and check the following PL pages:

* Action blocks demo pages
* Full Page Theming pages
* D8 Homepage pages
* D8 Events pages
